crossdock/main_test.go

```diff
@@ -95,6 +95,16 @@ func TestCrossdock(t *testing.T) {
 				"transport": []string{"http", "tchannel"},
 			},
 		},
+		{
+			name: "ctxpropagation",
+			axes: axes{
+				"transport": []string{"http", "tchannel"},
```

Comment:
Yeah, we criss-cross. `transport` specifies the transport we use to talk to the subject, and the other transport is used for the subject to talk to the client.